src/App.tsx

```diff
@@ -28,6 +29,7 @@ export function App({
           <ScrivitoNotFoundErrorPage>
             <NotFoundErrorPage />
           </ScrivitoNotFoundErrorPage>
+          <SiteNotFoundPage />
```

Comment:
* [ ] FYI It feels weird to have two components responsible for `*NotFound`. From a readers point of view the consequences of this can only be understood by reading both components. How about having a single component responsible for any 404 handling?